pkg/middleware/debug_trace.go

```diff
@@ -0,0 +1,33 @@
+package middleware
+
+import (
+	"net/http"
+
+	"go.opencensus.io/plugin/ochttp/propagation/b3"
+	"go.opencensus.io/trace"
+	"go.opencensus.io/trace/propagation"
+)
+
+const (
+	DebugTraceHeader = "X-Debug"
```

Comment:
it should be more explicit, sothing like `X-Expose-Tracing-ID`

Reply:
Does `X-Debug-Trace` make sense?